.github/workflows/go-test.yml

```diff
@@ -25,7 +40,14 @@ jobs:
       - uses: actions/setup-go@v5
         with:
           go-version: ${{ matrix.go-version }}
+      - name: Test Setup # optional test setup step
```

Comment:
I think I'm going to cast my vote for not having this here. That may mean that renterd can't use the global test workflow, but it might be better to split more of the logic into a reusable action where we can define multiple job steps instead. The MySQL tests can't run on Windows or Ubuntu anyway, so a generic setup step doesn't fit well.